packages/openops/src/lib/ai/providers/google-vertex.ts

```diff
@@ -0,0 +1,99 @@
+import { createVertex } from '@ai-sdk/google-vertex';
+import { createVertexAnthropic } from '@ai-sdk/google-vertex/anthropic';
+import { LanguageModel } from 'ai';
+import { AiProvider } from '../providers';
+
+const googleVertexGeminiModels = [
```

Comment:
idk if we should add these or not, but i figured if we add support might aswell add proper support